yb-voyager/cmd/importSchema.go

```diff
@@ -51,6 +51,7 @@ func init() {
 	importCmd.AddCommand(importSchemaCmd)
 	registerCommonImportFlags(importSchemaCmd)
 	registerImportSchemaFlags(importSchemaCmd)
+	target.Schema = strings.ToLower(target.Schema)
```

Comment:
@priyanshi-yb have you verify whether adding it here is working for `import schema` and `import data`
Because this is a parent command which we don't support now.

Reply:
I have added in specifically `importSchema` and `importData` init() functions and not in `import`'s init(). So, I believe it is fine and it working as well.

Reply:
ok